src/transformers/trainer.py

```diff
@@ -1635,18 +1636,26 @@ def optimizer_hook(param):
                 )
             if not is_accelerate_available("0.30.0"):
                 raise ImportError("You need to have `accelerate>=0.30.0` to be able to use schedulefree optimizers")
-            from schedulefree import AdamWScheduleFree, SGDScheduleFree
+            from schedulefree import AdamWScheduleFree, RAdamScheduleFree, SGDScheduleFree
```

Comment:
please add a msg to install schedulefree >=1.4 to use `RAdamScheduleFree` here. You need to create a `is_schedulefree_available` function just like what we did for accelerate above